packages/sdk/src/utils/tokenGateway.ts

```diff
@@ -129,8 +131,9 @@ export async function teleport(teleport_param: {
 	params: Params
 	apiPromise: ApiPromise
 	options: Partial<SignerOptions>
+	extrinsics?: Array<SubmittableExtrinsic<"promise", ISubmittableResult>>
```

Comment:
Add a test for this, you can batch a `system.remark` call with the teleport call